packages/storybook/stories/experiments/Helpers.mdx

```diff
@@ -0,0 +1,115 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Experiments/Helpers" />
+
+# Experiments / Helpers
+
+To make running experiments and loading feature flags easier across different pages, layouts and components in our codebase we have few helpers that you can use.
+
+## useFeature
+
+This hook is used to load and allocate feature flags and experiments. It is important to note that calling this hook automatically allocates the user in any matched experiment using requested feature flag.
+
+```tsx
+import { useFeature } from './GrowthBookProvider';
+import { feature } from '../lib/featureManagement';
+
+const MyComponent = () => {
```

Comment:
Don't know why but syntax highlighting does not work. It should work with `@storybook/addon-docs`. Any clue?

Reply:
Scratch that it looks to be working when built on preview, looks to be broken only locally.